python-flask-single-page-app/README.md

```diff
@@ -1,15 +1,25 @@
 # Python / Flask
 
-Simple Python API example for booting Intercom from a single page app. In this example, we just use jQuery to asynchronously load the user data to boot Intercom, but you can apply the same concept to any frontend framework.
+This repo is an example application of the Intercom Messenger on a Python single page app using Flask. You can follow along and look at how it is implemented in the code.
 
-For the purposes of demonstrating how the Intercom Messenger can be loaded differently for visitors and logged in users this app offers the option to create an account and login.
+On the **server side**, add an endpoint that will return identifying information about the currently logged in user to the Intercom Messenger.
+
+    *You can also return the user information in the login response instead of a dedicated endpoint if you prefer.*
+
+1. Add a GET route to your app called `/intercomSettings` ([see code](https://github.com/intercom/intercom-install-examples/blob/b61c0e4d1f3dd27a0bee3ee5688902355e48e348/python-flask-single-page-app/app/server/server.py#L13)).
+1. Add a method to the appropriate controller for the new route ([see code](https://github.com/intercom/intercom-install-examples/blob/b61c0e4d1f3dd27a0bee3ee5688902355e48e348/python-flask-single-page-app/app/server/server.py#L14)).
+1. Return an object to the client containing:
+   - The email of the currently logged user
+   - A hash based on the user’s email and your Intercom identity verification secret key like ([see code](https://github.com/intercom/intercom-install-examples/blob/b61c0e4d1f3dd27a0bee3ee5688902355e48e348/python-flask-single-page-app/app/server/server.py#L19)). If you want more info on why this is important, check out [this doc](https://docs.intercom.com/configure-intercom-for-your-product-or-site/staying-secure/enable-identity-verification-on-your-web-product).
+
+**On the client side**, connect to Intercom and load the Messenger:
+1. Paste the code right before the closing body tag of every page where you want the Intercom Messenger to appear like ([see code](https://github.com/intercom/intercom-install-examples/blob/b61c0e4d1f3dd27a0bee3ee5688902355e48e348/python-flask-single-page-app/app/static/index.html#L21)).
+
+*If you have already installed the Intercom Messenger on a page you want to track logged-in users on, remove the `window.intercomSettings` part as we will be using `window.Intercom(‘boot’, …)` to boot the Messenger.*
```

Comment:
Looks like this line needs indentation

![screen shot 2017-11-28 at 4 48 25 pm](https://user-images.githubusercontent.com/71852/33352073-0036109a-d45c-11e7-92e6-409b47333772.png)